server/src/main/java/org/spine3/server/command/CommandValidator.java

```diff
@@ -77,7 +77,7 @@ private CommandValidator() {
         result.addAll(messageViolations);
 
         final CommandContext context = command.getContext();
-        final String commandId = idToString(context.getCommandId()).trim();
```

Comment:
We have `Command/EventIdToStringConverter`s registered in the converter registry, so I think we can remove explicit command/event ID conversion from `idToString()`.